.github/workflows/release.yml

```diff
@@ -46,9 +46,18 @@ jobs:
             ${{ runner.os }}-go-
 
       - uses: sigstore/cosign-installer@e1523de7571e31dbe865fd2e80c5c7c23ae71eb4 # v3.4.0
+
+      - name: Create SBOM
+        uses: anchore/sbom-action@v0
```

Comment:
can we use a sha and specify version number in comment? For instance, see line 48 above